### ISSUE TITLE
Poetry could not find a pyproject.toml file

### ISSUE THREAD

**Issue by nwesterhausen** [Tiryoh/actions-mkdocs#58]
I get this error when running during an action:
```
  with:
    mkdocs_version: latest
    requirements: requirements.txt
    configfile: mkdocs.yml
/usr/bin/docker run --name e1cc512ae08094d696402e94a58f6f58ec36a7_9e57a1 --label e1cc51 --workdir /github/workspace --rm -e INPUT_MKDOCS_VERSION -e INPUT_REQUIREMENTS -e INPUT_CONFIGFILE -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_RUN_ATTEMPT -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_NAME -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/valheim-discordconnector/valheim-discordconnector":"/github/workspace" e1cc51:2ae08094d696402e94a58f6f58ec36a7  "latest" "requirements.txt" "mkdocs.yml"

  RuntimeError

  Poetry could not find a pyproject.toml file in /github/workspace or its parents

  at /root/.poetry/lib/poetry/_vendor/py3.8/poetry/core/factory.py:367 in locate
      363│             if poetry_file.exists():
      364│                 return poetry_file
      365│ 
      366│         else:
    → 367│             raise RuntimeError(
      368│                 "Poetry could not find a pyproject.toml file in {} or its parents".format(
      369│                     cwd
      370│                 )
      371│             )
```

**Comment by Tiryoh** [comment]
I guess the error has been fixed on #60, and this fix has released as v0.8.0.
Could you try with the new version? (Maybe re-running the action might be enough.)

**Comment by revjtanton** [comment]
It works for me now. Thanks @Tiryoh!
